O365/excel.py

```diff
@@ -1663,6 +1662,76 @@ def get_named_range(self, name):
         return self.named_range_constructor(parent=self, **{self._cloud_data_key: response.json()})
 
 
+class WorkbookApplication(ApiComponent):
+    _endpoints = {
+        'get_details': '/application',
+        'post_calculation': '/application/calculate'
+    }
+
+    def __init__(self, workbook_or_id, *, con=None, **kwargs):
```

Comment:
This should just take in workbook class as parent and use that main resource. No need for workbook_id